src/components/templates/ArtPiece/ArtPiece.tsx

```diff
@@ -56,47 +58,60 @@ export const ArtPiece: React.FC<ArtPieceProps> = ({ venue }) => {
     [customAspect]: true,
   });
 
+  const [validBannerImageUrl] = useValidImage(
+    venue?.config?.landingPageConfig?.coverImageUrl,
+    DEFAULT_VENUE_BANNER
+  );
+  const containerVars = useCss({
+    background: `url(${validBannerImageUrl})`,
+  });
+
+  const containerClasses = classNames("ArtPiece", containerVars);
+
```

Comment:
Why is this copy-pased in all the venues? Shouldn't a new hook `useBackground()` or something DRY it up?

Or even better, is it possible to have a `Background` component that can just be put into place (this would work even better than a hook) since there is already `<div className="ArtPiece-background" />` just waiting to be externalized (and take the image logic as its inside)

Reply:
created a new component `ContainerWithBackground` to be used in those places 65d922a

Reply:
@alexey-petrov maybe similar to `WithNavigationBar` this one can be named `WithVenueBackground`